### PR TITLE
Update tests to dabs and chart to helm3

### PR DESCRIPTION
**Pull request by piontec**
This PR does the following:

- upgrades the chart to be helm3-compatible
- tunes tests layout and marks to work with `dabs`

For some reason, validation with `ct` still doesn't work. I'm disabling it for now.

To run the build&test process, run from the main dir (with to-be-released tomorrow version of dabs.sh):

```
dabs.sh -c helm/efk-stack-app --skip-steps test_unit --destination build
```

### CODE REVIEW

File: helm/efk-stack-app/Chart.yaml [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
-#
-# or in the "license" file accompanying this file. This file is distributed
-# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
-# express or implied. See the License for the specific language governing
-# permissions and limitations under the License.
-
+annotations:
+  application.giantswarm.io/metadata: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/main.yaml
+  application.giantswarm.io/readme: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/README.md
+  application.giantswarm.io/values-schema: http://localhost/efk-stack-app-0.3.5-2d81bd191a40b9a3bf48936ab34ad65315a148ce.tgz-meta/values.schema.json
```
Comment (ubergesundheit) [inline helm/efk-stack-app/Chart.yaml:4]: Whats up with these links to `localhost`?

File: helm/efk-stack-app/.abs/main.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+replace-chart-version-with-git: true
```
Comment (ubergesundheit) [inline helm/efk-stack-app/.abs/main.yaml:1]: It this file correctly placed here? I thought it needs to be placed at the root of a repository? (`.abs/main.yaml`)

I'm running `dabs.sh` with container tag `0.1.2-7a6961a0338b59f11392a89a04baa621c226682d` in the root of the repository with flags `--steps all --chart-dir helm/efk-stack-app/` and it seems the configuration from there is not picked up..

Moving it to `.abs/main.yaml` works though.

File: helm/efk-stack-app/Chart.yaml (hunk 1 above)
Comment (piontec) [reply helm/efk-stack-app/Chart.yaml:4]: Someone hasn't changed catalog URL in the config ;)

**Comment by webwurst** [comment]
Ok, a bit messy here, but helm package is created and pushed. And I tested that once again on `ginger` and could see data in Kibana. Merging this now, let's see about how to switch to abs in circle-ci later.

**Pull request merged**